Sources/BraveWallet/WalletStrings.swift

```diff
@@ -1765,6 +1765,13 @@ extension Strings {
       value: "Not a valid SOL address",
       comment: "A warning that appears below the send crypto address text field, when the input `To` address is not a valid SOL address."
     )
+    public static let sendErrorDomainNotRegistered = NSLocalizedString(
+      "wallet.sendWarningDomainNotRegistered",
```

Comment:
Not sure if needed but var/key mismatch: `sendWarningDomainNotRegistered` → `sendErrorDomainNotRegistered`

Reply:
Nice catch thanks, I'll update this

Reply:
Fixed in [d89d046](https://github.com/brave/brave-ios/pull/6728/commits/d89d04697bd1f6ae5f7e70e1c398ab84e540db3f)